homeassistant/components/zha/sensor.py

```diff
@@ -231,7 +231,7 @@ def create_entity(
     def formatter(value: int) -> int | None:
         """Return the state of the entity."""
         # per zcl specs battery percent is reported at 200% ¯\_(ツ)_/¯
-        if not isinstance(value, numbers.Number) or value == -1:
```

Comment:
I wonder if the -1 was a left over from percentage number being reported as a signed 8 bit at some point, which would have resulted from interpreting 0xff as sint8. If so maybe that should be removed? Not that it hurts to keep.

Reply:
Yeah, I think we can remove `-1`. It's been this way since the initial implementation.